src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -154,6 +158,9 @@ const RenderListFeed = (props) => {
           hideThreeDot={true}
           props={item}
           height={getHeightHeader()}
+          isSelf={item?.is_self}
+          isFollow={item?.is_following_target}
+          onPressFollUnFoll={() => followUnfollowTopic(item, route?.params?.id, offset)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Here, the `onPressFollUnFoll` prop is being passed to the component. This prop is a function that calls `followUnfollowTopic`. Ensure that the `followUnfollowTopic` function has proper error handling to avoid unhandled promise rejections if the API call fails. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->